spec/rinject_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rinject'
+
+describe 'Inject method:' do
+
+  let(:arg)   { 10 }
+  let(:array) {[1,2,3,4]}
+
+  context 'Retesh Injects' do
+
+    it 'takes an initial argument' do
+      expect(array.retesh_inject(arg) { |a,b| a }).to eq array.inject(arg) { |a,b| a }
```

Comment:
Unused block argument - `b`. If it's necessary, use `_` or `_b` as an argument name to indicate that it won't be used.<br>Line is too long. [86/80]<br>Space missing after comma.